Sources/Paywalls/Components/PaywallButtonComponent.swift

```diff
@@ -167,6 +173,14 @@ public extension PaywallComponent {
             case inAppBrowser = "in_app_browser"
             case externalBrowser = "external_browser"
             case deepLink = "deep_link"
+
+            case unknown = "unknown"
+
+            public init(from decoder: Decoder) throws {
+                let container = try decoder.singleValueContainer()
```

Comment:
Should this default to `.unknown` if decoding fails instead of throwing the decoding error?

Reply:
Oh! Um... I didn't think of this 😅  But... we _probably_ won't make this anything other than a single string value 🤷‍♂️ That would super break things 😅 

If it was that drastic of a change, we'd probably make a whole new property since different object types but thanks for catching that anyway! 🫶